apps/site/src/views/Resources/components/ResourceCard/ResourceCard.module.scss

```diff
@@ -21,18 +21,28 @@
 	justify-content: center;
 	align-items: center;
 	gap: 10px;
-	height: 450px;
+	height: 350px;
 }
 
 .tag {
-	position: absolute;
-	left: -5%;
-	bottom: 5%;
+	background-image: url("~@/assets/images/resource-link.svg");
```

Comment:
Note: would personally prefer the native `BookmarkLink` approach but liking the more flexible background approach over the embedded text.